media-fonts/overpass/overpass-3.0.2.recipe

```diff
@@ -0,0 +1,39 @@
+SUMMARY="An open source web font family"
+DESCRIPTION="Inspired by the Highway Gothic font, designed by Delve Fonts"
+HOMEPAGE="http://overpassfont.org/"
+COPYRIGHT="Red Hat, Inc."
+LICENSE="SIL Open Font License v1.1"
+REVISION="1"
+SOURCE_URI="https://github.com/RedHatBrand/Overpass/releases/download/$portVersion/overpass-webfonts.zip"
+CHECKSUM_SHA256="8dad24f7b12d359dad64b1ce0c4c57e833ac2523d7411db12904acb3f6f98658"
+SOURCE_DIR="overpass-mono-webfont"
```

Comment:
I _think_ it would be worth inserting
```
SOURCE_FILENAME="overpass-webfonts-$portVersion.zip"
```
between CHECKSUM_SHA256 and SOURCE_DIR.
Otherwise HP will complain next time when a new archive becomes available.

> It comes with 2 top level directory, that's why theese changes required.

Regarding SOURCE_URI_2, I'm pretty sure you can workaround this (to avoid having to define SOURCE_URI_2) by setting
```
SOURCE_DIR=""
```